lsm-srlinux/ci/Jenkinsfile

```
@@ -73,15 +86,15 @@ pipeline {
     agent any
     
     parameters {
-        string(name: 'version', defaultValue: '6', description: 'Run the lsm quickstart against the specified version of the Python package of the ISO product.\nIf left empty it will run against iso6.\nThe version can be of format 6.1.1rc20230217144359, 6.1.1rc, 6.1.1 or 6')
+        string(name: 'version', defaultValue: '6dev', description: 'Run the lsm quickstart against the specified version of ISO product. The version can be of format 6.1.1rc20230217144359, 6.1.1rc, 6.1.1, 6 or 6dev.')
```

```suggestion
        string(name: 'version', defaultValue: '7dev', description: 'Run the lsm quickstart against the specified version of ISO product. The version can be of format 6.1.1rc20230217144359, 6.1.1rc, 6.1.1, 6 or 6dev.')
```
If we run against dev, doesn't 7dev make the most sense? Not too strong of an opinion, I think it all ties in with the more broader issue of what we should test against which versions. Ideally we'd test against all of them but we can't realistically make everything a matrix build... So we need to address this at some point but for now I'm fine with any of the logical choices.